src/aem/HISTORY.rst

```diff
@@ -2,6 +2,9 @@
 
 Release History
 ===============
+1.0.0
++++++
+* Remove msrestazure dependency

```

Comment:
Releasing it as 1.0.0 will assign a major version which has semantic versioning implications. Can we use `0.3.1`?

Reply:
According to https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md#initialization, `0.3.1` is not allowed. `1.0.0` is required